pytorch_lightning/plugins/training_type/training_type_plugin.py

```diff
@@ -42,12 +42,16 @@ class TrainingTypePlugin(ABC):
     loop."""
 
     def __init__(
-        self, checkpoint_io: Optional[CheckpointIO] = None, precision_plugin: Optional[PrecisionPlugin] = None
+        self,
+        accelerator: Optional["pl.accelerators.accelerator.Accelerators.accelerator.Accelerator"] = None,
```

Comment:
I legitimally laughed at this 😂😂😂

Did find+replace trick you?

Reply:
how is this possible ...

Reply:
```suggestion
        accelerator: Optional["pl.accelerators.accelerator.Accelerator"] = None,
```